.github/dependabot.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+updates:
+  - package-ecosystem: bundler
+    directory: /
+    schedule:
+      interval: daily
+    allow:
+      # Security updates
+      - dependency-name: brakeman
+        dependency-type: direct
+      # Internal gems
+      - dependency-name: "govuk*"
+        dependency-type: direct
+      - dependency-name: "rubocop-govuk"
+        dependency-type: direct
+      - dependency-name: "gds-sso"
+        dependency-type: direct
+      - dependency-name: govspeak
+        dependency-type: direct
+      - dependency-name: plek
+        dependency-type: direct
+      - dependency-name: pg
+        dependency-type: direct
```

Comment:
I don't think you meant postgres to be an internal gem.

Reply:
I just realised this and was going back to change it - good spot! Will move the comment.

Reply:
Ah ha, sorry - you nearly got away with it.